src/webportal/src/app/user/fabric/userView/index.jsx

```diff
@@ -0,0 +1,265 @@
+                },
+              });
+            }, 100);
+          });
+      },
+    });
+  };
+
+  const editUser = (user) => {
+    showEditInfo(user.username, user.admin, user.virtualCluster, user.hasGithubPAT);
+  };
+
+  const showEditInfo = (username, isAdmin, vcList, hasGithubPAT) => {
+    $('#modalPlaceHolder').html(userEditModalComponent({
+      'username': username,
+      'isAdmin': String(isAdmin),
+      'vcList': vcList,
+      'hasGithubPAT': String(hasGithubPAT),
+      updateUserVc,
+      updateUserAccount,
+      updateUserGithubPAT,
+    }));
+    $('#userEditModal').modal('show');
+  };
+
+  const updateUserInfoCallback = (data) => {
+    if (data.error) {
+      showMessageBox(data.message);
+    } else {
+      showMessageBox({
+        text: 'Update user information successfully',
+        dismissedCallback: () => {
+          $('#userEditModal').modal('hide');
+          setAllUsers([]);
+          refreshAllUsers();
+        },
+      });
+    }
+  };
+
+  const updateUserVc = (username) => {
+    const virtualCluster = $('#form-update-virtual-cluster :input[name=virtualCluster]').val();
+    updateUserVcRequest(username, virtualCluster)
+      .then(updateUserInfoCallback)
+      .catch((err) => {
+        $('#form-update-virtual-cluster').trigger('reset');
+        showMessageBox(err);
+      });
+  };
+
+  const updateUserAccount = (username) => {
+    const password = $('#form-update-account :input[name=password]').val();
+    const admin = $('#form-update-account :input[name=admin]').is(':checked') ? true : false;
+    updateUserAccountRequest(username, password, admin)
+      .then(updateUserInfoCallback)
+      .catch((err) => {
+        $('#form-update-account').trigger('reset');
+        showMessageBox(err);
+      });
+  };
+
+  const updateUserGithubPAT = (username) => {
+    const githubPAT = $('#form-update-github-token :input[name=githubPAT]').val();
+    updateUserGithubPATRequest(username, githubPAT)
+      .then(updateUserInfoCallback)
+      .catch((err) => {
+        $('#form-update-github-token').trigger('reset');
+        showMessageBox(err);
+      });
+  };
+
+  window.updateUserVc = updateUserVc;
+  window.updateUserAccount = updateUserAccount;
+  window.updateUserGithubPAT = updateUserGithubPAT;
+
+  const context = {
+    allUsers,
+    refreshAllUsers,
+    filteredUsers,
+    ordering,
+    setOrdering,
+    filter,
+    setFilter,
+    pagination,
+    setPagination,
+    setSelectedUsers,
+    getSelectedUsers,
+    setAllSelected,
+    addUser,
+    importCSV,
+    removeUsers,
+    editUser,
+  };
+
+  return (
+    <Context.Provider value={context}>
+      <Fabric style={{height: '100%'}}>
+        <Stack verticalFill styles={{root: {position: 'relative'}}}>
+          <Stack.Item>
+            <TopBar />
+          </Stack.Item>
+          <Stack.Item grow styles={{root: {height: 1, overflow: 'auto', backgroundColor: 'white', paddingTop: 15}}}>
```

Comment:
We are moving to a fabric based design system now.
- Spacing: get the spacing value from fabric's theme object. [example](https://github.com/microsoft/pai/blob/0eb286f029d78865c24feb6aabcc990c44336c11/src/webportal/src/app/home/home.jsx#L67)
- Color: use fabric's ColorClassNames.
- Font: use fabric's FontClassNames.

Reply:
Good suggestion. Fixed.